### ISSUE TITLE
Migration to ngrx 15 not included in nx 15.3.0

### ISSUE THREAD

**Issue by yusijs** [nrwl/nx#13689]
### Current Behavior

Running `nx migrate @nrwl/workspace@latest` creates no migrations, and leaves ngrx-packages at 14.*

### Expected Behavior

Running `nx migrate @nrwl/workspace@latest` creates ngrx-migrations, and updates ngrx-packages to 15.*

### Steps to Reproduce

1. Run `nx migrate @nrwl/workspace@latest` in any repo currently at <15.3.0

### Nx Report

```shell
❯ nx report

 >  NX   Report complete - copy this into the issue template

   Node : 16.18.1
   OS   : darwin x64
   npm  : 8.19.2
   
   nx : 15.2.1
   @nrwl/angular : 15.2.1
   @nrwl/cypress : 15.2.1
   @nrwl/detox : Not Found
   @nrwl/devkit : 15.2.1
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 15.2.1
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 15.2.1
   @nrwl/js : 15.2.1
   @nrwl/linter : 15.2.1
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : 15.2.1
   @nrwl/nx-cloud : 15.0.2
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/rollup : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 15.2.1
   @nrwl/web : Not Found
   @nrwl/webpack : 15.2.1
   @nrwl/workspace : 15.2.1
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
         @ngrx/component-store: 14.0.2
         @ngrx/effects: 14.0.2
         @ngrx/entity: 14.0.2
         @ngrx/router-store: 14.0.2
         @ngrx/store: 14.0.2
         @ngrx/schematics: 14.0.2
         @ngrx/store-devtools: 14.0.2
```

### Additional Information

Relevant pr: https://github.com/nrwl/nx/pull/13498
Release notes: https://github.com/nrwl/nx/releases/tag/15.3.0

**Comment by markgoho** [comment]
try `npx nx migrate @ngrx/store`

**Comment by leosvelperez** [comment]
Thanks for reporting this!

The issue is that the package update was set to target Nx v15.2.4-beta.0, but the change was actually released on Nx v15.3.0. Therefore, if you were already in v15.2.4, the migrations that are set to target that version won't run again when migrating to Nx v15.3.0.

Note this only affects workspaces on Nx v15.2.4 and it won't affect workspaces on an older version than Nx v15.2.4. I'll push a fix so the migration targets the next patch version.

For those who were already in Nx v15.2.4 and are facing this issue now and can't wait for the next patch version, you can run the following:

```shell
nx migrate @ngrx/store@latest
```

**Comment by leosvelperez** [comment]
@yusijs interesting, I'll have a look.
